gcc_plugin/afl-gcc-rt.o.c

```diff
@@ -138,8 +138,9 @@ static void __afl_map_shm(void) {
 
 static void __afl_start_forkserver(void) {
 
-  static u8 tmp[4];
-  s32       child_pid;
+  u8  tmp[4] = {0, 0, 0, 0};
```

Comment:
this could be initialized with `{0}` but for 4 chars it doesn't matter either way